docs/pipelines/tinycap.md

```diff
@@ -72,6 +72,32 @@ One can specify other options such as: `bowtie` (alignment) options, clipping an
 }
 ```
 
+## Taxonomy extraction 
```

Comment:
Maybe your right for now indeed. What we should have is a MultiSampleMapping page that explaining all the common features. Maybe we should make a structure like this:
```
- pipelines
  - multisamplemapping
    - Shiva
    - Gentrap
    - Carp
  - Other pipelines
```

Will make a new issue to do this in the future